doc/design/scheduled-tasks.md

```diff
@@ -0,0 +1,55 @@
+# Asynchronous Tasks
+We are currently (7/16/2021) using celery to run these asynchronous tasks:
+
+ * Calculating blob sha256 digests
+ * Writing dandiset.yaml and assets.yaml after publishing
+ * Validating asset metadata
+ * Validating dandiset metadata
```

Comment:
these processes should be more or less instantaneous. perhaps we do want to split out "assets Summary generation" as a non instantaneous task?

Reply:
I'm not following your suggestion. Tasks can be long-running, but there's no problem with having an instantaneous task. Why would we want to split `assetsSummary` generation into a separate task, and how would that change the implementation?

Reply:
i wasn't sure why these are celery tasks. they seem to be completable within the duration of a REST call and be done without too much load. perhaps it's the semantics again, but i'm seeing these as calls to dandischema.validate, which does not need to be asyncrhonous.

Reply:
I see. I think that refactoring can be done later, outside of this design doc.